modules/gitops-release-notes-1-3-0.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-3_{context}"]
+= Release notes for {gitops-title} 1.3
+
+{gitops-title} 1.3 is now available on {product-title} 4.7, 4.8, and 4.9.
+
+[id="new-features-1-3_{context}"]
+== New features
+In addition to the fixes and stability improvements, the following sections highlight what is new in {gitops-title} 1.3.0:
+
+* After the  Operator is installed, the Red Hat Single Sign On (SSO) is automatically configured. You can log into the default Argo CD instance in the `openshift-gitops` namespace using the OpenShift credentials. As an admin you can disable the Red Hat SSO installation after the Operator is installed which will result in removing the Red Hat SSO deployment from the `openshift-gitops` namespace.
+Additionally, for any new Argo CD instance you can enable the Red Hat SSO by setting the flag to true. When the flag is not present, the Red Hat SSO will be disabled by default.
+* The default Argo CD instance installed by the  Operator as well as accompanying controllers can now run on the infrastructure nodes of the cluster by setting a simple configuration toggle.
+* Internal communications in Argo CD can now be secured using the TLS and the OpenShift cluster certificates. The Argo CD routes can now leverage the OpenShift cluster certificates in addition to using external certificate managers such as the cert-manager.
+* Use the improved *environment details* view in the *Developer* perspective of the console 4.9 to gain insights into the GitOps environments.
+* You can now access to custom health checks in Argo CD for `DeploymentConfig` resources, `Route` resources, and `OLM` Operators.
```

Comment:
Please remove 'to' from: You can now access to custom health checks

Reply:
@saumeya @jaideepr97 @reginapizza the only mention of OLM-Operator is here:https://docs.openshift.com/container-platform/4.9/security/certificate_types_descriptions/olm-certificates.html
My question hence is should this be written as `olm-operator`, and if it should be singular or plural as in operator/operators.
Also in this context is `Route` a custom resource?
I wonder if this should be written as:
* You can now access custom health checks in Argo CD for `DeploymentConfig` resources, routes, and `OLM-Operators`.
@DebarghoGhosh  please change depending on answers to the above question.

Reply:
@saumeya please suggest

Reply:
@praveenkumar, @chetan-rns  please suggest

Reply:
I believe `olm-operator` has a different meaning since it is one of the OLM components. Here "OLM Operators" refers to the operators installed using OLM. And Route is an OpenShift custom resource.

Reply:
Thank you, Chetan. In that case @DebarghoGhosh we simply need to replace `OLM` Operators with 'operators installed using OLM.'

Reply:
Hey Debargho, @praveenkumar is our CRC rockstar :) We might need either Praveen Thangdancha or another QE from the team to provide an LGTM before we merge.